app/inpututils.cpp

```diff
@@ -1961,6 +1964,11 @@ QUrl InputUtils::iconFromGeometry( const Qgis::GeometryType &geometry )
   }
 }
 
+void InputUtils::sanitizeFileName( QString &fileName )
+{
+  fileName.replace( ILLEGAL_FILENAME_CHARS, QStringLiteral( "_" ) );
```

Comment:
Can we also trim the `fileName`?

Reply:
This is slightly tricky!
Should we trim before the file extension? After? Both?
We're using this method to also sanitize the Default Path, which doesn't have an extension, but if it has a `.` we should not trim before it!